tx/textual/valuerenderer/message.go

```diff
@@ -0,0 +1,79 @@
+package valuerenderer
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"sort"
+	"strings"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+type messageValueRenderer struct {
+	tr *Textual
+}
+
+var _ ValueRenderer = (*messageValueRenderer)(nil)
+
+func (mr *messageValueRenderer) Format(ctx context.Context, v protoreflect.Value, w io.Writer) error {
+	fds := v.Message().Descriptor().Fields()
+	fields := make([]protoreflect.FieldDescriptor, 0, fds.Len())
+	for i := 0; i < fds.Len(); i++ {
```

Comment:
Please swap the names of these variables

Reply:
Done :D